src/allmydata/test/common_util.py

```diff
@@ -57,9 +55,10 @@ def __setitem__(self, key, value):
         return
 
 def insecurerandstr(n):
-    return ''.join(map(chr, map(randrange, [0]*n, [256]*n)))
+    return b''.join(map(bchr, map(randrange, [0]*n, [256]*n)))
```

Comment:
If this (and other?) functions are now ported to Python 3 then perhaps it makes sense to move them to `common_py3` and then import them into this module for compatibility?

Reply:
Given this module is now Python 3 compatible, the other approach is to get rid of `common_py3` and merge it back into this module; it was only a temporary step to have a utilities module that imported on Python 3, it doesn't need to exist anymore.

Reply:
Ah right.  That sounds fine, yes.  Your comment makes me realize that when I was thinking about this I was conflating "ported to Python 3", which this module is not, with "Python 3 compatible", which this module now must be.

Reply:
Ticketed as https://tahoe-lafs.org/trac/tahoe-lafs/ticket/3456.